examples/gallery/maps/choropleth_map.py

```diff
@@ -14,45 +14,36 @@
 """
 
 # %%
-import geodatasets
 import geopandas as gpd
 import pygmt
 
-# Read the example dataset provided by geodatasets.
-gdf = gpd.read_file(geodatasets.get_path("geoda airbnb"))
-print(gdf.head())
+provider = "https://naciscdn.org/naturalearth/"
+world = gpd.read_file(f"{provider}50m/cultural/ne_50m_admin_0_countries.zip")
+world["POP_EST"] *= 1e-6
```

Comment:
Need to use `.copy()` to avoid the warning:
```
/home/docs/checkouts/readthedocs.org/user_builds/pygmt-dev/conda/4228/lib/python3.13/site-packages/geopandas/geodataframe.py:1968: SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  super().__setitem__(key, value)
```
```suggestion
world["POP_EST"] *= 1e-6
africa = world[world["CONTINENT"] == "Africa"].copy()
```